src/v2/providers/alerts/alerts.ts

```diff
@@ -70,7 +70,7 @@ export type AlertType =
   | "crashlytics.velocity"
   | "crashlytics.newAnrIssue"
   | "billing.planUpdate"
-  | "billing.automatedPlanUpdate"
+  | "billing.planAutomatedUpdate"
```

Comment:
Is the backend changed to this?

Reply:
Oh it was me! - go/eafxg

Reply:
yep. we actually have it right in the billing.ts.